Dockerfile

```diff
@@ -251,6 +251,20 @@ ARG NPM_MARKDOWN_TABLE_FORMATTER_VERSION=1.6.1
 # renovate: datasource=github-tags depName=skaji/cpm
 ARG PERL_PERLCRITIC_VERSION=0.997021
 
+# renovate: datasource=composer depName=squizlabs/php_codesniffer
+ARG PHP_SQUIZLABS_PHP_CODESNIFFER_VERSION=3.11.3
+# renovate: datasource=composer depName=bartlett/sarif-php-converters
+ARG PHP_BARTLETT_SARIF_PHP_CONVERTERS_VERSION=1.1.0
+# renovate: datasource=composer depName=phpstan/phpstan
+ARG PHP_PHPSTAN_PHPSTAN_VERSION=2.1.8
+# renovate: datasource=composer depName=phpstan/extension-installer
+ARG PHP_PHPSTAN_EXTENSION_INSTALLER_VERSION=1.4.3
+# renovate: datasource=composer depName=vimeo/psalm
+ARG PHP_VIMEO_PSALM_VERSION=6.8.8
+# renovate: datasource=composer depName=overtrue/phplint
+ARG PHP_OVERTRUE_PHPLINT_VERSION=9.5.6
+# renovate: datasource=composer depName=friendsofphp/php-cs-fixer
+ARG PHP_FRIENDSOFPHP_PHP_CS_FIXER_VERSION=v3.71.0
```

Comment:
is it intended that it is the only one that has a "v" in it?

Reply:
It is the only one that explicitly includes it in packagist: https://packagist.org/packages/friendsofphp/php-cs-fixer

Reply:
In a repo of mine, the composer.json file is:
![image](https://github.com/user-attachments/assets/9256d9a2-823a-42e8-8afb-32a4a169f20e)
Where I didn't really change the versions there, it is renovate that handles it.

Reply:
As I said in another comment, I have left the version as it was (8.3)